### PR TITLE
Fix checkstyle errors, mostly final vars

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

**Comment by davidxia** [comment]
@rohansingh @gimaker

File: checkstyle-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_0.dtd">
+
+<suppressions>
+  <suppress files="src/test/.*" checks="FinalLocalVariable"/>
```
Comment (gimaker) [inline checkstyle-suppressions.xml]: why suppress rather than fix these?

**Comment by gimaker** [comment]
:+1:

File: checkstyle-suppressions.xml (hunk 1 above)
Comment (davidxia) [inline checkstyle-suppressions.xml]: Maybe @rohansingh didn't care much about test sources? I might as well fix these too?

Comment (mattnworb) [inline checkstyle-suppressions.xml]: +1 for treating src/main and src/test the same

**Comment by davidxia** [comment]
My second commit also fixes it for tests.

**Comment by gimaker** [comment]
:+1:

**Pull request merged**